R/prob.Overlap.r

```diff
@@ -11,13 +11,16 @@
 #' information.
 #' @return A matrix of three probabilities.
 #' @details
-#' When creating the density plot, it is important to increase the \code{n.grid}
+#' When creating the density plot from overlap, it is important to increase the \code{n.grid}
```

Comment:
Looks like this has overlap instead of activity written in the text?

Reply:
I left the overlap stuff in there. So, can use activity or overlap